subsys/net/ip/ipv6.h

```diff
@@ -205,7 +205,11 @@ static inline int net_ipv6_finalize(struct net_pkt *pkt,
 #if defined(CONFIG_NET_IPV6_MLD)
 int net_ipv6_mld_join(struct net_if *iface, const struct in6_addr *addr);
 #else
-#define net_ipv6_mld_join(...)
+static inline int
+net_ipv6_mld_join(struct net_if *iface, const struct in6_addr *addr)
+{
+	return 0;
```

Comment:
Shouldn't the stub return an error code (`-ENOTSUP`)?

Reply:
Depends on what you want the code to do in that case. I assumed the idea behind the previous variadic macros was to silently do nothing. Do you prefer it to fail?

Reply:
I don't think it should be hidden from the user that the feature is not implemented/enabled. If the application expects it to work (i. e. return success on return), then enable the feature, otherwise handle the error code accordingly.

Reply:
Changed